mvnw

```diff
@@ -1,316 +0,0 @@
-#!/bin/sh
-# ----------------------------------------------------------------------------
-# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This shouldnt be deleted?

Reply:
Honestly don't know why that happened, reverted